test/ui-test/testSuites/suite_communities/tst_communityFlows/test.feature

```diff
@@ -11,133 +11,104 @@
-        Then the user lands on the community named myCommunity
-        When the admin edits the current community to the name <new_community_name> and description <new_community_description> and color <new_community_color>
-        When the admin goes back to the community
-        Then the user lands on the community named <new_community_name>
-
+	@mayfail
+	# TODO: Weak. Sometimes passes sometimes fails (standalone or as part of the sequence). To anaylize.
+	# Also TODO: Missing validation of description changed and color changed. Now validation only checks the new community name.
+    Scenario Outline: The admin edits a community
+        When the admin renames the community to "<new_community_name>" and description to "<new_community_description>" and color to "<new_community_color>"
+        And the admin goes back to the community
+        Then the user lands on the community named "<new_community_name>"
         Examples:
             | new_community_name       | new_community_description  | new_community_color |
             | myCommunityNamedChanged  | Cool new description 123   | #ff0000             |
 
-
-	@merge
-    Scenario: User leaves community
-        When the user creates a community named testCommunity, with description My community description, intro Community Intro and outro Community Outro
-        Then the user lands on the community named testCommunity
-        When the user opens app settings screen
-        And the user opens the communities settings
-        And the user leaves the community
-        Then the count of communities in navbar is 0
-
-
-	@merge @mayfail
-    Scenario Outline: User changes the emoji of a channel
-        When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
-        Then the user lands on the community named myCommunity
-        When the admin creates a community channel named test-channel, with description My description with the method bottom_menu
-        Then the user lands on the channel named test-channel
-        When the user changes emoji of the current community channel with emoji by description <new_emoji_description>
-        Then the community channel has emoji <new_emoji>
-
+	@mayfail
+	# TODO: Test broken. Validation doesn't work (No attribute ` emoji`found error).
+    Scenario Outline: The admin changes the emoji of a channel
+        When the admin changes the current community channel emoji to "<new_emoji_description>"
+        Then the community channel has emoji "<new_emoji>"
         Examples:
             | new_emoji_description | new_emoji |
             | thumbs up             | 👍        |
 
+    # TODO: This scenario must be in a different feature since it does not accomplishe the start/en sequence and / or background
+    # Add new test case that contains scenarios related to create/delete and navigate throw communities and usage of navbar.
```

Comment:
Should we move it now or add an issue to readd it later?

Reply:
Oh yeah! I forgot to add this particular task. Here it is: https://github.com/status-im/status-desktop/issues/8034
Thanks for pointing it out!